src-doc/contenuti/requisiti.tex

```diff
@@ -0,0 +1,160 @@
+
+        \hline
+        RF\_08 & Obbligatorio & Il sistema deve riportare l'intensità luminosa dell'area al valore di default una volta passato il tempo impostato & Capitolato, UC \\ %uc corrispondenti non chiare?%
+
+        \hline
+        RF\_09 & Obbligatorio & L'utente deve effettuare l'accesso per poter utilizzare le funzionalità del sistema & Capitolato, UC01\\
+
+        \hline
+        RF\_10 & Obbligatorio & L'utente deve poter inserire la locazione geografica dell'area di illuminazione & Capitolato, UC10\\
+
+        \hline
+        RF\_11 & Obbligatorio & L'utente deve poter inserire il raggio d'azione dell'area di illuminazione & Capitolato, UC\\ %manca uc?%
+
+        \hline
+        RF\_12 & Obbligatorio & L'utente deve essere in grado di visualizzare quali aree sono illuminate in un dato momento & Capitolato, UC2\\
+
+        \hline
+        RF\_13 & Obbligatorio & L'utente deve poter impostare la modalità dell'area di illuminazione come manuale & Capitolato, UC21\\
+
+        \hline
+        RF\_14 & Obbligatorio & L'utente deve poter impostare la modalità dell'area di illuminazione come automatica & Capitolato, UC22\\
+
+        \hline
+        RF\_15 & Obbligatorio & L'utente deve poter modificare la luminosità dei lampioni del sistema a livello globale & Capitolato, UC23\\
+
+        \hline
+        RF\_16 & Opzionale & Il sistema deve essere in grado di regolare automaticamente l'intensità luminosa dei lampioni in caso rilevi valori fuori soglia di luminosità ambientale & Capitolato\\
+
+        \hline
+        RF\_17 & Opzionale & Il sistema deve essere in grado di rilevare automaticamente guasti nei lampioni & Capitolato \\
+
+        \hline
+        RF\_18 & Obbligatorio & L'utente deve essere in grado di segnalare manualmente guasti nei lampioni & Capitolato, UC6 \\
+
+        \hline
+        RF\_19 & Opzionale & Il gestore deve poter visualizzare l'elenco degli impianti guasti & Capitolato, UC07 \\
+
+        \hline
+        RF\_20 & Obbligatorio & L'utente deve essere in grado di rimuovere guasti dalla lista & Capitolato, UC9 \\
+
+        \hline
+        RF\_20 & Opzionale & Il sistema deve essere in grado di regolare automaticamente l'intensità luminosa dei lampioni in caso rilevi valori fuori soglia di luminosità ambientale & Capitolato\\
+
+        \hline
+        RF\_21 & Obbligatorio & Il sistema deve essere in grado di ricevere informazioni dal sensore in modalità Push & Capitolato \\
+
+        \hline
+        RF\_22 & Opzionale & Il sistema deve essere in grado di ricevere informazioni dal sensore in modalità Pull & Capitolato \\
+
+        \hline
+        RF\_23 & Opzionale & Il sistema visualizzare un messaggio di errore in caso di eccezioni & VIN\_20221129 \\ % non so se metterlo: credo sia corretto ma andrebbe creata l'uc. %
```

Comment:
Qui va un po' definita meglio la questione, è un po' troppo vaga

Reply:
Tecnicamente credo andrebbe definito un requisito per ogni situazione dove potrebbe saltare fuori un messaggio d'errore...però mi sa diventano parecchie uc in più.